modules/seqexec/server/src/main/scala/seqexec/server/gmos/GmosControllerEpics.scala

```diff
@@ -363,78 +374,81 @@ object GmosControllerEpics extends GmosEncoders {
                      ccParams(state.cc, config.cc) ++
                      nsParams(state.ns, config.ns)
 
-        IO(Log.info("Start Gmos configuration")) *>
-          IO(Log.debug(s"Gmos configuration: ${config.show}")) *>
+        L.info("Start Gmos configuration") *>
+          L.debug(s"Gmos configuration: ${config.show}") *>
           warnOnDHSNotConected *>
           (params.sequence *>
             sys.configCmd.setTimeout[IO](ConfigTimeout) *>
             sys.post
           ).unlessA(params.isEmpty) *>
-          IO(Log.info("Completed Gmos configuration"))
+          L.info("Completed Gmos configuration")
       }
 
-      override def setRowsToShuffle(rows: Int): IO[Unit] =
-        IO(Log.debug(s"Set rows to shuffle to $rows")) *>
-          DC.setNsRows(rows) *>
-          sys.configCmd.setTimeout[IO](ConfigTimeout) *>
-          sys.post.void
-
       override def observe(fileId: ImageFileId, expTime: Time): IO[ObserveCommandResult] =
         failOnDHSNotConected *>
           sys.observeCmd.setLabel(fileId) *>
           sys.observeCmd.setTimeout[IO](expTime + ReadoutTimeout) *>
-          sys.observeCmd.post[IO]
+          sys.observeCmd.post[IO].flatMap {
+            case ObserveCommandResult.Paused =>
+              sys.nsState.map(GmosEncoders.nsStateDecoder.decode).map {
```

Comment:
@jluhrs This worked fine in test but there maybe a little risk given that GMOS is automagically changing this to `CLASSIC`. WDYT?

Reply:
Yes, but it changes to CLASSIC after the detector readout. At that point the observation state cannot change to PAUSED.